src/processor/operator/persistent/reader/csv/parallel_csv_reader.cpp

```diff
@@ -30,32 +33,43 @@ bool ParallelCSVReader::hasMoreToRead() const {
 
 uint64_t ParallelCSVReader::parseBlock(block_idx_t blockIdx, DataChunk& resultChunk) {
     currentBlockIdx = blockIdx;
+    rowsInCurrentBlock = 0;
     seekToBlockStart();
     if (blockIdx == 0) {
         readBOM();
         if (option.hasHeader) {
-            readHeader();
+            uint64_t headerNumRows = readHeader();
+            errorHandler->setHeaderNumRows(headerNumRows);
         }
     }
     if (finishedBlock()) {
         return 0;
     }
     ParallelParsingDriver driver(resultChunk, this);
-    return parseCSV(driver);
+    const auto numRowsParsed = parseCSV(driver);
```

Comment:
Remove const since numRowsParsed is a primitive type.

Reply:
Wondering what is the rationale for not having const on primitive types?